tests/test-git.t

```diff
@@ -1,17 +1,44 @@
   $ git init foobar
   Initialized empty Git repository in .* (re)
   $ cd foobar
+  $ git config --local filter.nbstripout.extrakeys ' '
   $ echo -n "*.txt text" >> .git/info/attributes
   $ ${NBSTRIPOUT_EXE:-nbstripout} --is-installed
   [1]
   $ ${NBSTRIPOUT_EXE:-nbstripout} --install
   $ ${NBSTRIPOUT_EXE:-nbstripout} --is-installed
-  $ git diff --no-index --no-ext-diff --unified=0 --exit-code -a --no-prefix ${TESTDIR}/test_diff.ipynb ${TESTDIR}/test_diff_output.ipynb
-  $ git diff --no-index ${TESTDIR}/test_diff.ipynb ${TESTDIR}/test_diff_different.ipynb
+  $ git diff --no-index ${TESTDIR}/test_diff.ipynb ${TESTDIR}/test_diff_output.ipynb
+  [1]
```

Comment:
I don't know why I need the `[1]` as the diff is empty.

Reply:
Have you tried add `--exit-code`? What's the reason for removing the test above?

Reply:
Sorry, I can see you reverted this.

Reply:
I replaced line 9 with 10-11, like I said I don't know why

Reply:
reverted, seems to work fine. Just my machine